sdk/program/src/blake3.rs

```diff
@@ -0,0 +1,218 @@
+//! The `blake3` module provides functions for creating SHA-256 hashes.
```

Comment:
SHA-256 hashes?

Reply:
Ah, ok, thanks!

Reply:
That comment is inaccurate.  Blake3 and Keccak were _SHA3_ contenders. Keccak having won, is known as SHA3, it's 256bit digest variant as SHA3-256.  SHA256 reference the SHA2 standard 256bit digest.  Blake3 is just Blake3

Reply:
Thanks!